cdc/sorter/leveldb/system/system_test.go

```diff
@@ -16,7 +16,12 @@ package system
 import (
 	"context"
 	"testing"
+	"time"
 
+	"github.com/pingcap/tiflow/cdc/sorter/leveldb"
+	"github.com/pingcap/tiflow/cdc/sorter/leveldb/message"
+	"github.com/pingcap/tiflow/pkg/actor"
+	actormsg "github.com/pingcap/tiflow/pkg/actor/message"
```

Comment:
how aboout name it `actorMsg`. `actormsg` would make my linter annoying.

Reply:
I prefer `actormsg`, like some std package names, `ioutil`, `tzdata`, `expvar`, etc.